data_adapter_oemof/build_datapackage.py

```diff
@@ -92,29 +98,36 @@ def __split_timeseries_into_years(parametrized_sequences):
         return split_dataframes
 
     @staticmethod
-    def get_foreign_keys(
-        process_busses: list, mapper: Mapper, components: list
-    ) -> list:
+    def get_foreignKeys(struct: list, mapper: Mapper, components: list) -> list:
```

Comment:
I wasn't sure what to use here either. Since as you mentioned in the datapackage the the `foreignKeys` are in camel case. I'll change to normal Variable name.